app/models/event.rb

```diff
@@ -5,4 +5,39 @@ class Event
     :application, :scenario, :controller, :action, :timestamp, :event_type, :content, :duration,
     :position, :started_at, :finished_at
   )
+
+  class << self
+    # {
+    #   "content": null,
+    #   "duration": 0.138118,
+    #   "event_type": "read_fragment.action_controller",
+    #   "finished_at": 1440509586.217381,
+    #   "position": 90,
+    #   "started_at": 1440509586.217243
+    # },
+    def find_by(application:, event_source:, scenario:)
+      query = Druid::Query::Builder.new
+      query
+      .group_by([:position, :event_type])
+      .granularity(:all)
+      .filter(application: application.id)
+      .filter(event_source: event_source.id)
+      .filter(scenario: scenario.events_hash)
+      .long_sum(:count)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.